db/db.go

```
@@ -0,0 +1,35 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/daniyalumer/todo-list-go-chi/conf"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+var Db *sql.DB
+
+func Connect() error {
+	dsn := fmt.Sprintf("host=%s port=%s user=%s dbname=%s password=%s sslmode=disable",
+		conf.DbHost,
+		conf.DbPort,
+		conf.DbUser,
+		conf.DbName,
+		conf.DbPassword,
+	)
+
+	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
```

err is not handled here